itest/starboard/starboard_cli_test.go

```diff
@@ -1014,10 +1014,29 @@ var _ = Describe("Starboard CLI", func() {
 
 	Describe("Command scan ciskubebenchreports", func() {
 
+		It("scan wrong node name. It doesn't create CISKubeBenchReports", func() {
```

Comment:
When you merge Describe and It it should read as the whole sentence:

*Command scan ciskubebenchreports should not create CISKubeBenchReport when node does not exist.*

```suggestion
		It("should not create CISKubeBenchReport when node does not exist", func() {
```